### PR TITLE
Add Windows support

### LINKED ISSUE #4: Get it working on Windows.
In theory, this should be straightforward because Symbolic supports PDB.

### PR DESCRIPTION
**Pull request by nnethercote**
Fixes #4 

r? @gabrielesvelto

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -156,21 +158,44 @@ impl Fixer {
...
+                    // PE files should contain a pointer to a PDB file.
+                    let pdb_file_name = pe.debug_file_name().ok_or_else(|| msg("find PDB for"))?;
+                    Some(pdb_file_name.to_string())
+                } else {
+                    panic!(); // Impossible: peek() said it was a PE object.
+                }
+            }
+            _ => None,
+        };
+        if let Some(file_name2) = file_name2 {
```
Comment (gabrielesvelto) [inline src/main.rs:228]: Move this to a separate function, so that you  can write something more natural like:
```
if let Some(file_name2) = MaybeGetExternalDebugInfo() {
  [...]
}
```

File: src/main.rs [hunk 2]
```diff
@@ -156,21 +158,44 @@ impl Fixer {
...
-        // Build the `FileInfo` from the debug session.
+        // Build the `FileInfo` from the debug session. `tests/README.md` has an
+        // explanation of the commented-out `eprintln!` statements.
         let mut interner = Interner::default();
-        let func_infos = debug_session
+        let mut func_infos: Vec<_> = debug_session
             .functions()
             .filter_map(|function| {
                 let function = function.ok()?;
-                // This eprintln! is useful when creating tests.
```
Comment (gabrielesvelto) [inline src/main.rs:173]: Why did you remove this line?

File: src/main.rs [hunk 3]
```diff
@@ -185,7 +210,6 @@ impl Fixer {
                         .lines
                         .into_iter()
                         .map(|line| {
-                            // This eprintln! is also useful when creating tests.
```
Comment (gabrielesvelto) [inline src/main.rs:188]: Ditto

File: tests/README.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+The unit tests refer to specific addresses within the generated binaries. These
+addresses were determined for each generated executable by uncommenting the
+`eprintln!` statements in `Fixer::build_file_info` and running `fix-stacks` on
+an input that mentions that executable. For example, the debug info for
+`example-linux` was obtained using an input file containing this line:
+```
+#00: ???[tests/example-linux +0x0]
+```
+The resulting `FUNC` and `LINE` lines can be seen in `src/tests.rs`.
+
```
Comment (gabrielesvelto) [inline tests/README.md]: This is great, thanks!

**Review by gabrielesvelto — APPROVED** [review]
LGTM with just a few nits

File: src/main.rs (hunk 2 above)
Comment (nnethercote) [reply src/main.rs:173]: I moved the comment to the top of the loop, to avoid the comment duplication. And it now points to the `tests/README.md` file.

File: src/main.rs (hunk 1 above)
Comment (nnethercote) [reply src/main.rs:228]: I tried that but it didn't feel good. I had to duplicate the `msg` closure, and the return type of the new function was `Result<Option<String>, String>` which is pretty gross.

Instead I split off the second half of this function into `FileInfo::new`. That keeps all the fallible I/O operations in this function, while moving the infallible `FileInfo` construction into a `FileInfo` method. That's a much nicer way to break this function into smaller pieces.
